website/docs/r/ecs_service.html.markdown

```diff
@@ -69,7 +69,7 @@ into consideration during task placement. The maximum number of
 Load balancers support the following:
 
 * `elb_name` - (Required for ELB Classic) The name of the ELB (Classic) to associate with the service.
-* `target_group_arn` - (Required for ALB) The ARN of the ALB target group to associate with the service.
+* `target_group_arn` - (Required for modern Load Balancers) The ARN of the Load Balancer target group to associate with the service.
```

Comment:
Can you change this to `(Required for ALB/NLB)` please? Amazon's definition of "modern" might change in the future.

Reply:
You got it!